docs/git-comparison.md

```diff
@@ -17,10 +17,13 @@ various use cases.
 * **The working copy is automatically committed.** That results in a simpler and
   more consistent CLI because the working copy is now treated like any other
   commit. [Details](working-copy.md).
-* **There's no index (staging area).** That also results in a simpler
-  CLI for similar reasons. The index is very similar to an intermediate commit
-  between `HEAD` and the working copy, so workflows that depend on it can be
-  modeled using proper commits instead. [Details](#the-index).
+* **There's no index (staging area).** Because the working copy is automatically
```

Comment:
Is there a better title for this item?

Reply:
Maybe try: 

> No separate, explicit staging area (the "index").

Reply:
I was thinking more along the lines of "We have something better to replace the staging area" but I'm not sure how to say it succinctly enough.

Reply:
My suggestion
> Maybe no intermediate saved state (no staging area)

If we take Wol's quote on LWN to the extreme. 

> By saying "an index does not make sense", you're not saying that you don't acknowledge that it solves a problem. You're simply saying that the git approach doesn't work for jujutsu.

Reply:
I think the name is fine as is. Another option:

> The index (AKA staging area) is replaced with regular commits.